source/extensions/filters/http/ext_proc/http_client/http_client_impl.cc

```diff
@@ -0,0 +1,45 @@
+#include "source/extensions/filters/http/ext_proc/http_client/http_client_impl.h"
+
+#include "source/common/common/enum_to_int.h"
+#include "source/common/http/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+void ExtProcHttpClient::onSuccess(const Http::AsyncClient::Request&,
```

Comment:
I think these functions here will be updated when real implementation comes in.

Reply:
Right